cmd/kyma/alpha/create/module/module.go

```diff
@@ -273,12 +276,39 @@ func (cmd *command) Run(ctx context.Context) error {
 		archiveFS = memoryfs.New()
 		l.Info("using in-memory archive")
 	}
-	// this builds the archive in memory, Alternatively one can store it on disk or in temp folder
-	archive, err := module.CreateArchive(archiveFS, cmd.opts.ModuleArchivePath, modDef)
-	if err != nil {
-		cmd.CurrentStep.Failure()
-		return err
+
+	var archive *comparch.ComponentArchive
+	gitPath, err := files.SearchForTargetDirByName(modDef.Source, ".git")
+	if gitPath == "" || err != nil {
+		l.Warnf("could not find git repository root, using %s directory", modDef.Source)
+		n := nice.NewNice()
+		n.PrintImportant("! CAUTION: The target folder is not a git repository. The sources will be not added to the layer")
+		if files.IsFileExists(cmd.opts.SecurityScanConfig) {
+			n.PrintImportant("  The security scan configuration file was provided. But the security scan will be skipped")
```

Comment:
```suggestion
			n.PrintImportant("  The security scan configuration file has been provided, but it will be skipped due to the absence of repository information.")
```